ECHO-COVID19.ipynb

```diff
@@ -388,7 +219,7 @@
    },
    "source": [
     "## Are facilities being inspected?\n",
-    "Stack tests may not be happening, but regulators could go in and inspect facilities.\n",
+    "Stack tests may not be happening, but regulators could go in and inspect facilities. Are they?\n",
```

Comment:
"Stack tests may not be happening, but regulators could go in and inspect facilities. Are they conducting inspections?\n".

Reply:
added